src/components/container/PartnerBanner/PartnerBanner.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './PartnerBanner.scss';
+import Logo from './assets/partnerLogos/prgh-dark-main.svg';
+
+function PartnerBanner() {
+  const banners = Array(10).fill({ name: "PRGH", src: Logo });
+
+  return (
+    <div className='PartnerBanner'>
+      <div>
+        <img alt="start" />

```

Comment:
what are these two img-tags without src used for?

Reply:
for testing purposes, to see the starting banner and ending banner

Reply:
Okay, then remove please. The code we merge into mian should be production ready